site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -220,18 +221,13 @@ const getDefaultBuildParameters = (
   return buildValues;
 };
 
-export const orderedTemplateParameters = (
-  templateParameters?: TemplateVersionParameter[],
-): TemplateVersionParameter[] => {
-  if (!templateParameters) {
-    return [];
-  }
-
-  const immutables = templateParameters.filter(
-    (parameter) => !parameter.mutable,
-  );
-  const mutables = templateParameters.filter((parameter) => parameter.mutable);
-  return [...immutables, ...mutables];
+export const orderTemplateParameters = (
```

Comment:
I didn't see this used anywhere in the codebase, but one of the problems with the previous function was that even if the immutable and mutable parameters were grouped together, there wasn't a quick way to determine where one group stopped, and the next started